opensearch_py_ml/ml_commons/ml_common_client.py

```diff
@@ -77,3 +78,135 @@ def load_model(self, model_id: str):  # type: ignore
             method="POST",
             url=API_URL,
         )
+
+    def get_task_info(self, task_id: str):  # type: ignore
```

Comment:
How about just name the method as `get_task`?
Same comment for `get_model_info`, change to `get_model`?

Reply:
Can we remove `# type: ignore` ?